clr-k8s-examples/create_stack.sh

```diff
@@ -10,6 +10,8 @@ SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
 : ${MASTER_IP:=}
 HIGH_POD_COUNT=${HIGH_POD_COUNT:-""}
 
+CNI=${CNI:-"canal"}
+RUNNER=${RUNNER:-"crio"}
```

Comment:
I thought this change was to moved to a different PR. Is that not true? Also the runner settings should happen in `setup_system.sh` .

Reply:
good catch. i thought so too

Reply:
ok so that was in system_setup.sh https://github.com/clearlinux/cloud-native-setup/pull/181

Reply:
Guess we do need RUNNER here because of cilium. but maybe it should reflect current containerd setting in the repo and #181 should update it to crio even for create_system.sh

Reply:
So Runner is set to crio, because we are using Crio as a default based on 181 change, and the i used these here so that the user can specify to use CNI to either be Canal, Cilium, Flannel.

Reply:
Ok depending on the ordering of which one to merge first you can figure this out